modules/ztp-image-based-upgrade-installing-oadp.adoc

```diff
@@ -13,7 +13,7 @@ Install and configure the OADP Operator with {ztp} before starting the upgrade.
 
 .Procedure
 
-. Create the following CRs in the `openshift-adp` namespace and push them to the `source-crs/custom-crs` directory:
+. Create the following CRs in the `openshift-adp` namespace for both: the seed and target clusters, then push the CRs to the `source-crs/custom-crs` directory:
```

Comment:
I don't think there should be a colon here

```suggestion
. Create the following CRs in the `openshift-adp` namespace for both seed and target clusters, then push the CRs to the `source-crs/custom-crs` directory:
```

Reply:
@opayne1 Thanks, updated! To answer your question, @achuzhoy is the QE reviewing this PR :)

Reply:
Wonderful! Thank you!